.github/workflows/main.yml

```diff
@@ -543,6 +411,47 @@ jobs:
             src/libzenroom_dll.lib
             src/zenroom.dll
 
+  changes:
```

Comment:
About this job:
* id should be more meaningfull, like `python-changes`
* `name` is missing, helpfull for a better understading looking at the CI result
* this action should be performed only for commits, add dependency to semantic release with release equal to True and remove `permissions`
* There is only one output, but it is not present in `outputs`, remove backend/fronted and add instead `python3`
* Remove comments